util/Ozone_Monitor/nwprod/oznmon_shared/sorc/oznmon_horiz.fd/oznmon_read_diag.f90

```diff
@@ -629,7 +629,7 @@ subroutine read_ozndiag_data_nc( ftin, header_fix, data_fix, data_nlev, data_ext
     !
     !  The newer netcdf read processes each field for all obs 
     !  so it can grab all the obs data in a single call.  The
-    !  calling routine in time.f90 uses the iflag value to 
+    !  calling routine in horiz.f90 uses the iflag value to 
```

Comment:
`time` and `horiz` are the only differences between `oznmon_read_diag.f90` in `oznmon_horiz.fd` and `oznmon_time.fd`.   `time` and `horiz` are used correctly in each instance.   Might it be better to generalize the language here so that both source code files are identical?  Eventually the ozone diagnostic file reader could be placed in a library and invoked by both programs.   Ensuring the two copies of `oznmon_read_diag.f90` are identical is a step in this direction.

Reply:
Change has been made.